src/hooks/index.ts

```diff
@@ -702,3 +704,18 @@ export const useLogout = () => {
     dispatch(signout())
   }
 }
+
+export const useRefreshOnFocusHack = () => {
```

Comment:
move this out of here
this is for cross platform hooks only

Reply:
done